content/installation/java/installation/AWS.md

```diff
@@ -0,0 +1,47 @@
+<!--
+title: "Installing on AWS"
+description: "AWS installation process"
+tags: "java agent installation AWS ebs elasticbeanstalk"
+-->
+
+Amazon Web Services (AWS) is a secure cloud services platform that provides a reliable, scalable and affordable computing platform solution. By using AWS, Contrast users can maximize the security of their applications.
+
+Contrast works with the following AWS systems:
+
+* AWS Elastic Beanstalk
+* Amazon EC2
+* Amazon RDS
+
+## Before You Start
+
+* [Download and install the Java agent](installation-java.html#java-standard) from Contrast.
+* Create an [AWS Elastic Beanstalk account](https://aws.amazon.com/elasticbeanstalk/).
+
+## Installation
+
+### Step one
+
+* In your WAR file, add the *contrast.jar* file to *WEB-INF/lib/*.
```

Comment:
@meg23 @dhafley Whoa...this is really weird. Is this the only way to get our agent into that environment?

Reply:
This is the easiest way without talking through building an AMI